packages/flutter/lib/src/widgets/title.dart

```diff
@@ -11,12 +11,15 @@ import 'framework.dart';
 /// A widget that describes this app in the operating system.
 class Title extends StatelessWidget {
   /// Creates a widget that describes this app to the operating system.
+  ///
+  /// If [title] is null, then the title will be set to the empty string.
   Title({
```

Comment:
const

Reply:
Do you mean you want Title to be a const constructor?  It can't be, because the color.alpha in the assertion isn't.